app/components/EntryVerification/index.tsx

```diff
@@ -10,7 +10,7 @@ import { useModalState } from '#hooks/stateManagement';
 import { useLazyRequest } from '#base/utils/restRequest';
 import UserContext from '#base/context/UserContext';
 
-import { EntryAction } from '../constants';
+import { EntryAction } from '#components/commentConstants';
```

Comment:
Why not have this inside Comment? Do we need a "commentContacts" component?

Reply:
we use those values elsewhere too.